### PR TITLE
Sending draft status in output from PL Editor #2677

### PR DESCRIPTION
**Pull request by poppashingles**
I had to delete the package-lock file and rebuild it, so it's updated a lot of the dependencies. All 25 tests are still passing ok on my local version.

### CODE REVIEW

File: dist/PublicLab.Editor.js [hunk 1]
```diff
@@ -22132,35 +22132,36 @@ module.exports = PublicLab.Formatter = Class.extend({
...
+
+        output.title              = data.title || null;
+        output.body               = data.body  || null;
+
         // we can remove this from server req, since we're authenticated
-        output.authenticity_token = data.token || null; 
- 
+        output.authenticity_token = data.token || null;
+        // Whether note will be draft or not
+        output.draft = data.draft || false;
```
Comment (Gauravano) [inline dist/PublicLab.Editor.js]: Awesome !!! how about introducing new lines aroud these 2 lines as it's looking bit of messy. Thanks @poppashingles .

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (Gauravano) [inline package-lock.json]: Can't say about this. Let's ask @jywarren about it.

File: dist/PublicLab.Editor.js (hunk 1 above)
Comment (poppashingles) [reply dist/PublicLab.Editor.js]: New lines added, thanks!

**Comment by jywarren** [comment]
Yes, `package-lock.json` is not needed -- do you think you could remove it? Also, if it's possible to make these changes to the files in `/src/` rather than `/dist/` -- basically `dist` files are auto-generated from `src` files, so this change would be come overwritten, does that make sense? 

But looking great, thank you both!

**Comment by poppashingles** [comment]
Thanks @jywarren :) I've removed the package-lock and moved the code into the PublicLab.Formatter.js file in src/adapters. Thanks again!

File: src/adapters/PublicLab.Formatter.js [hunk 3]
```diff
@@ -16,35 +16,38 @@ module.exports = PublicLab.Formatter = Class.extend({
...
+        output.title              = data.title || null;
+        output.body               = data.body  || null;
+
         // we can remove this from server req, since we're authenticated
-        output.authenticity_token = data.token || null; 
- 
+        output.authenticity_token = data.token || null;
+
+        // Whether note will be draft or not
+        output.draft = data.draft || false;
```
Comment (Gauravano) [inline src/adapters/PublicLab.Formatter.js]: Awesome @poppashingles !!!

File: package-lock.json (hunk 2 above)
Comment (Gauravano) [reply package-lock.json]: Its still there, could you please try one more time to remove it

Comment (poppashingles) [reply package-lock.json]: Hiya. In my forked one the package lock is gone and the pull request seems to be saying everything in it is gone. Am I reading it wrong?

**Comment by Gauravano** [comment]
@poppashingles how about opening a PR just by adding lines in the concerned file from Github so that we don't get any extra additions and deletions? What do you think ?

**Comment by Gauravano** [comment]
hi @poppashingles , I have resolved the conflict with the `package-lock.json` file. Thanks for your help. Also, you have made changes in the master branch of your PR, that's a wrong way. The right way(from my knowledge) is to make a new branch, then make changes and commit to that branch and final step is raise a PR.

**Comment by Gauravano** [comment]
hey @jywarren its ready now. Please review it. thanks.

**Comment by poppashingles** [comment]
Thanks guys. Sorry it's not gone particularly smoothly! Been a really good learning experience though so thank you :)

**Comment by Gauravano** [comment]
No issue @poppashingles, we all are here for learning and I will also try to open more clear FTO from next time. Thanks for the help! Also, if you like to solve more issues, you can find them here https://publiclab.github.io/community-toolbox/#r=all

**Comment by jywarren** [comment]
Hi, all - thanks so much @poppashingles and @Gauravano for taking this one on! I think it just needs one more run of `grunt build` to get the final changes from `/src/` to be compiled into `/dist/` properly. I don't see them in the changes (they should match the changes from `/src/`, so if one of you could give that a try, I think that should be our last step!

Again, great work and thank you!

**Comment by Gauravano** [comment]
hi @jywarren , I have run the `grunt build`. please review it again. Thanks.

**Comment by jywarren** [comment]
Awesome, merging now then!

**Comment by jywarren** [comment]
For this to show up on publiclab.org we now need to bump the version number by a `0.x.0` increment (see semver.org) and create a new release.

**Comment by jywarren** [comment]
Ok, did that! Now we need to grab the latest version on publiclab.org with
`bower update` - you should be able to do this locally to test!

**Comment by Gauravano** [comment]
Hey, @jywarren thanks for merging this. I have checked locally, changes are in place.

**Pull request merged**